configs/sst_cs_system_management-backup-tools.yaml

```diff
@@ -24,6 +24,14 @@ data:
   - xorriso
   - libisoburn
   - libisoburn-doc
+
+  arch_packages:
+  #rear not yet supported on s390x and arm
+
+  x86_64:
```

Comment:
You miss one level of indentation here.

(yay, YAML!)

Reply:
Sorry, you are right...

Reply:
Thanks!